src/html/index.html

```diff
@@ -0,0 +1,180 @@
+<!DOCTYPE html>
+<html lang="ru">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+    <link rel="stylesheet" href="assets/css/main.css">
+</head>
+<body>
+    <header>
+        <div class="container">
+            <div class="header">
+                <img src="./assets/images/logo1.png" alt="Икит Сфу" class="header__logo">
+                <nav class="nav">
+                    <ol class="header__navigation site-navigation">
+                        <li class="site-navigation__el site-navigation--color-secondary site-navigation--size-m">
+                            <a href="#">
+                                Главная
+                            </a>
+                        </li>
+                        <li class="site-navigation__el site-navigation--color-secondary site-navigation--size-m">
+                            Информация
+                        </li>
+                        <li class="site-navigation__el site-navigation--color-secondary site-navigation--size-m">
+                            Новости
+                        </li>
+                        <li class="site-navigation__el site-navigation--color-secondary site-navigation--size-m">
+                            Контакты
+                        </li>
+                    </ol>
+                </nav>
+                <div class="header__user user-info">
+                    <div class="text--color-light text--size-l">
+                        Ваше имя
+                    </div>
+                    <img src="./assets/images/Avatar.png" alt="Аватар" class="user-info__photo">
+                </div>
+            </div>
```

Comment:
тут также, text--color-light text--size-l это все должно принадлежать блоку, не могут модификаторы висеть просто так, без блока или элемента

поэтому тут так
```suggestion
                <div class="header__user user-info">
                    <div class="user-info__name">
                        Ваше имя
                    </div>
                    <img src="./assets/images/Avatar.png" alt="Аватар" class="user-info__photo">
                </div>
            </div>
```